dips/dip-10.md

```diff
@@ -13,47 +14,37 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-* Sender enters receiver's Diem ID and other relevant information into their wallet application
-* Sender's wallet attempts to fetch the receiver's profile picture and full name along with a confirmation dialog
-* Sender confirms the identity of the recipient
-* Sender presses ‘send’ button
-* Receiver's account is credited the amount in the transaction
-* Receiver's wallet displays recent transactions and their senders
+# Diem ID Format
+Diem defines globally unique identifiers to facilitate transacting on the Diem Payment Network. As Diem stores no PII on-chain, these identifiers are exchanged during the off-chain process between two VASPs to identify a distinct source and destinations within their respective VASPs in order to produce a reference_id that can be stored on-chain. The format for the identifiers follows:
+`[user_identifier]@[vasp_domain_identifier]`
 
-# Diem ID format
-Diem ID is a string in format `<user-id>$<diem-id-domain>`.
+Example: `alice@avasp`
 
-Example: `andrey$novi`
-
-* **user-id**, identifies an account within a VASP:
+* `user_identifier` is a unique identifier meaningful only to a VASP that represents either the source or destination of a transaction. In practice, they are likely to be re-used and should be capable of both being the sender and recipient in a transaction. Their specific lifetime is undefined but should ideally be meaningful to the owner of the identifier. Specification:
   * Case insensitive
-  * Regex: `^[a-zA-Z0-9][a-zA-Z0-9-_+.]*$`
+  * Valid regular expression: ^[a-zA-Z0-9][a-zA-Z0-9-_\.]*$
   * Maximum length: 64 characters
-  * Stored internally in the VASP. Not published on the chain, there is no way for an external entity to list all the users within VASP.
-  * Defined by the VASP - user selected, auto-generated, etc.
-* **diem-id-domain** identifies a VASP:
+* `vasp_domain_identifier` is a well-known string that can be mapped to a VASP. Specification:
   * Case insensitive
-  * Regex: `^[a-zA-Z0-9][a-zA-Z0-9-]+$`
+  * Valid regular expression: ^[a-zA-Z0-9][a-zA-Z0-9-_\.]*$
   * Maximum length: 63 characters
-  * Association between diem id and VASP is published on the chain 
 
-## Diem ID domains
-Each Diem ID domain is associated with a single VASP. This information is stored on the block chain in a `DiemIdDomains` resource published in a parent VASP's on-chain account.
+
+# Naming Service (Off-Chain Lookup Service)
+The Diem ID specifies that VASPs can receive payments at a vasp_domain_identifier. The intent is to eventually store these values on-chain. As an initial effort into this space, the Diem Association stores a file mapping domain name to account address. The account address should be a parent VASP. The VASP sending a payment can then obtain the DualAttestation::Credentials::base_url to perform the ReferenceID exchange. An example CSV is:
+```
+avasp, 0xf72589b71ff4f8d139674a3f7369c69b
+bvasp, 0xc5ab123458df0003415689adbb47326d
+```
```

Comment:
would recommend we put this at the end of the on-chain, mentioning that initial efforts will leverage a CSV file until the on-chain data is ready